students/elmar_m/lesson04/kata14.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+'''
+file: kata14.py
+elmar_m / 22e88@mailbox.org
+Lesson04: Kata Fourteen: Tom Swift Under Milk Wood
+'''
+
+import collections
+import random
+
+
+def cleanup(x):
+    '''
+    Substitute certain characters in a string by whitespace. 
+    Return the cleaned string. 
+    
+        ARGS:
+    x: a string object
+    '''
+    x = x.replace('-', ' ')
+    x = x.replace('.', ' ')
+    x = x.replace(',', ' ')
+    x = x.replace('"', ' ')
+    x = x.replace("'", ' ')
+    x = x.replace(":", ' ')
+    x = x.replace('?', ' ')
+    x = x.replace('!', ' ')
+    x = x.lower()
+    return x 
+
+
+def make_oneline(x):
+    '''
+    Remove line breaks from a file. 
+    Return the files content as a string respectively one long line.
+    
+        ARGS:
+    x: the file to operate on
+    ''' 
+    with open(x, 'r') as myfile:
+        l = ' '.join(l.strip() for l in myfile)
+        return l
```

Comment:
a bit more pythonic to return in place -> `return ' '.join(l.strip() for l in myfile)` (especially since the vars used are non-descriptive)